client/src/main/scala/drt/client/components/Layout.scala

```diff
@@ -31,30 +32,39 @@ object Layout {
         <.div({
           val model = modelProxy()
 
-            model.airportConfig.renderReady { airportConfig =>
-              model.user.renderReady { user =>
+          model.airportConfig.renderReady { airportConfig =>
+            model.user.renderReady { user =>
+              <.div(
+                <.div(^.className := "topbar",
+                  <.div(^.className := "main-logo"),
+                  EnvironmentWarningComponent(),
+                  <.div(^.className := "alerts", AlertsComponent())
+                ),
                 <.div(
-                  <.div(^.className := "topbar",
-                    <.div(^.className := "main-logo"),
-                    EnvironmentWarningComponent(),
-                    <.div(^.className := "alerts", AlertsComponent())
-                  ),
                   <.div(
-                    <.div(
-                      Navbar(props.ctl, props.currentLoc.page, user, airportConfig),
-                      <.div(^.className := "container",
-                        <.div(<.div(props.currentLoc.render()))
-                      ),
-                      VersionUpdateNotice()
-                    )
+                    Navbar(props.ctl, props.currentLoc.page, user, airportConfig),
+                    <.div(buildFeedBackNavBar(user)),
+                    <.div(^.className := "container",
+                      <.div(<.div(props.currentLoc.render()))
+                    ),
+                    VersionUpdateNotice()
                   )
                 )
-              }
+              )
             }
+          }
         })
       }
     })
     .build
 
+  def buildFeedBackNavBar(user: LoggedInUser): VdomTagOf[UList] = {
+    <.ul(^.className := "nav navbar-nav navbar-right",
+      <.li(<.div(<.span(^.className := "btn", "is this page useful ?", ^.disabled := true))),
```

Comment:
I think this should be "Is this page useful?" 

i.e. start with a capital letter, and not have a space before the question mark, but I don't know if that was a design choice?